Flaky test in LongPollingActivateJobsTest

**Describe the bug**
Flaky test in LongPollingActivateJobsTest

**Log/Stacktrace**
```
[2019-08-06T09:50:51.024Z] [ERROR]   LongPollingActivateJobsTest.shouldUnblockAllRequestsWhenJobsAvailable:126 

[2019-08-06T09:50:51.024Z] activateJobsStub.handle(<any>);

[2019-08-06T09:50:51.024Z] Wanted 48 times:

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.job.LongPollingActivateJobsTest.shouldUnblockAllRequestsWhenJobsAvailable(LongPollingActivateJobsTest.java:126)

[2019-08-06T09:50:51.024Z] But was 24 times:

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.024Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)

[2019-08-06T09:50:51.025Z] -> at io.zeebe.gateway.api.util.StubbedGateway$StubbedBrokerClient.sendRequest(StubbedGateway.java:121)
```
**Environment:**
 - OS: [e.g. Linux]
 - Zeebe Version: [e.g. 0.20.0]
 - Configuration: [e.g. exporters etc.]

--- a/gateway/src/test/java/io/zeebe/gateway/api/job/ActivateJobsStub.java
+++ b/gateway/src/test/java/io/zeebe/gateway/api/job/ActivateJobsStub.java
@@ -16,8 +16,8 @@ import io.zeebe.gateway.impl.broker.response.BrokerResponse;
 import io.zeebe.protocol.Protocol;
 import io.zeebe.protocol.impl.encoding.MsgPackConverter;
 import io.zeebe.protocol.impl.record.value.job.JobBatchRecord;
-import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.LongStream;
 import org.agrona.DirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
@@ -43,7 +43,7 @@ public class ActivateJobsStub
       new UnsafeBuffer(MsgPackConverter.convertToMsgPack(CUSTOM_HEADERS));
   public static final DirectBuffer VARIABLES_MSGPACK =
       new UnsafeBuffer(MsgPackConverter.convertToMsgPack(VARIABLES));
-  private Map<String, Integer> availableJobs = new HashMap<>();
+  private Map<String, Integer> availableJobs = new ConcurrentHashMap<>();
 
   public long getJobBatchKey() {
     return JOB_BATCH_KEY;

